linode/manager
# Remove Launch Darkly Patch

## Description

- Free to Fail, but might be able to merge if all is working well
- Putting this up just to run tests against it for now

After further looking, I'm pretty sure the patch hasn't been needed since the repo has been converted away from [lerna](https://github.com/lerna/lerna)

## How to test

- Have Launch Darkly open and Cloud open at the same time
- Toggle some feature flags and ensure you see changes in your Cloud Manager
- Ensure Launch Darkly still works as expected
